tests/test_wave_04.py

```diff
@@ -73,6 +79,7 @@ def test_get_highest_word_tie_prefers_ten_letters_unsorted_list():
     assert best_word[0] == "AAAAAAAAAA"
     assert best_word[1] == 18
 
+#@pytest.mark.skip()
```

Comment:
Since the changes in your test files are just to comment out the skip test decorate, you don't technically need to add and commit these changes (but it doesn't hurt anything to do so). You could just add and commit the changes in game.py and that would make your pull request more concise with just the logical changes you made.